### DISCUSSION TITLE
add a white icon

### DISCUSSION THREAD

**Discussion by yyt666666** [File-New-Project/EarTrumpet#1323] [Ideas]
在Windows11中，软件图标在任务栏中为黑色，不容易看清，建议增加白色图标

In Windows11，the software icon is black in the taskbar, which is not easy to see. It is recommended to add a white icon

![QQ截图20230220232234](https://user-images.githubusercontent.com/47512829/220146250-fe207a2a-16bb-4c62-9c70-6b442907db51.png)

**Comment by shoulders** [comment]
When you use a dark theme, a dark theme in retrobar you cannot see the EarTrumpet icon

![image](https://github.com/user-attachments/assets/44f29595-3022-40be-b150-2f186514238b)

**Reply by riverar** [reply]
cc: @dremin

**Reply by dremin** [reply]
In RetroBar Properties > Advanced, there are settings for inverting icons, which is intended for this purpose. Try this: Set "invert system icons" to "always", then click "Customize", uncheck "Invert" for all icons and check it for EarTrumpet only.

**Reply by shoulders** [reply]
I can confirm that I can invert the EarTrumpet icon (to white) using the following settings as outlined above:

![image](https://github.com/user-attachments/assets/a45bfa84-9f9c-4929-824d-2eecf85c4aea)

`When needed by theme` option does not work for me. My theme is `Dark Mode Windows XP`

![image](https://github.com/user-attachments/assets/ead38097-90c1-4108-921f-3e3f53f38f4e)

Thanks for the help
